src/common/components/DataTable/FilterDropdown.tsx

```diff
@@ -166,6 +194,16 @@ export const FilterDropdown: FC<{
     else setOpenFilter(filter);
   };
 
+  const checkIfActiveFilter = (filter: FilterInfo): string => {
+    return activeFilters.filter(a => a.attr === filter.attribute).length ? 'active' : '';
```

Comment:
[`Array.find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) is probably the more appropriate function here.